### ISSUE TITLE
No key file found for extension

### ISSUE THREAD

**Issue by lfuelling** [nccgroup/autochrome#28]
Hi, 

as someone else mentioned in [this comment](https://github.com/nccgroup/autochrome/pull/19#issuecomment-567022249) to #19, the installation of prebuilt `.crx` files in the `data/extensions` folder fails with: 

```
No key file found for extension /path/to/autochrome/data/extensions/extension.crx
```

It would be nice to at least have a workaround for this described somewhere.

**Comment by lfuelling** [comment]
It seems like the only tiime the key is used is to write it into the `manifest.json` at `lib/auto_chrome/chrome_extension.rb:22`.

I'll try adding a check if the `manifest.json` already contains the key property in which case the `.pub` file might not be needed and will create a PR in case it works.

**Comment by peter-h-ncc** [comment]
Closed, merged into #42
